moduleframework/mtf.yaml

```diff
@@ -0,0 +1,104 @@
+# default module type in case you don't wath everytime use MODULE=docker or define it in config.yaml
+default_module: docker
+
+# openshift specific section
+openshift:
+  init_wait: 50
```

Comment:
Configuration file is a bit long. What about to create a directory `/etc/mtf/` and have there several configuration files. One for each type. Ease for modification.

Reply:
this is good idea, but I'm not sure if anybody than me will modify it :-)
We can discuss it. I like this idea